InternalDocs/garbage_collector.md

```diff
@@ -578,11 +613,9 @@ tracking status of the object.
       False
       >>> gc.is_tracked([])
       True
-      >>> gc.is_tracked({})
+      >>> gc.is_tracked(("a": 1))
```

Comment:
That's not valid syntax — did you mean to write a tuple?

```suggestion
      >>> gc.is_tracked(("a", 1))
```

Reply:
Yes I did. thanks.